app/javascript/dashboard/routes/dashboard/inbox/components/InboxCard.vue

```diff
@@ -143,21 +212,50 @@ export default {
     },
     snoozedUntilTime() {
       const { snoozed_until: snoozedUntil } = this.notificationItem;
-      return snoozedUntil;
+      if (!snoozedUntil) {
+        return null;
+      }
+      const timestamp = snoozedReopenTimeToTimestamp(snoozedUntil);
+      return shortenSnoozeTime(this.dynamicTime(timestamp));
+    },
+    hasLastSnoozed() {
+      return this.notificationItem?.meta?.last_snoozed_at;
     },
     snoozedDisplayText() {
+      if (this.hasLastSnoozed) {
+        return this.$t('INBOX.LIST.SNOOZED_ENDS');
+      }
       if (this.snoozedUntilTime) {
-        return `${this.$t('INBOX.LIST.SNOOZED_UNTIL')} ${snoozedReopenTime(
-          this.snoozedUntilTime
-        )}`;
+        return this.$t('INBOX.LIST.SNOOZED_UNTIL', {
+          time: this.shortTimestamp(this.snoozedUntilTime),
+        });
       }
       return '';
     },
   },
+  mounted() {
+    this.$nextTick(() => {
+      this.resizeObserver = new ResizeObserver(
+        debounce(this.resizeObserverCallback, OBSERVER_DEBOUNCE_TIME, false)
+      );
+      if (this.$refs.inboxCardInfo) {
+        this.resizeObserver.observe(this.$refs.inboxCardInfo);
+      }
+    });
+  },
   unmounted() {
+    if (this.resizeObserver) {
+      this.resizeObserver.unobserve(this.$refs.inboxCardInfo);
+      this.resizeObserver.disconnect();
+    }
     this.closeContextMenu();
   },
   methods: {
+    resizeObserverCallback(entries) {
+      entries.forEach(entry => {
+        this.inboxCardInfoElementWidth = entry.contentRect.width;
+      });
```

Comment:
Instead of looping over this, we can just take the first entry

```
      const entry = entries[0];
      if (entry) {
        this.inboxCardInfoElementWidth = entry.contentRect.width;
      }
```

Reply:
Resolving the comment https://github.com/chatwoot/chatwoot/pull/9109/commits/af90e3a56cff138db408a698d1a5a21b69072263